src/daemon/emitter.cc

```diff
@@ -579,6 +608,27 @@ void Emitter::DoRemoteExecute(const base::WorkerPool& pool, ResolveFn resolver,
     failed_tasks_->Push(std::move(*task));
     counter.ReportOnDestroy(true);
   }
+
+  auto new_conf = this->conf();
```

Comment:
I have to verify that we can guarantee the configuration is already reloaded at this point